molecule/verifier/lint/ansible_lint.py

```diff
@@ -0,0 +1,84 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```

Comment:
Should  be `... Red Hat ...`, please check other recently edited files.

Reply:
```suggestion
#  Copyright (c) 2019 Red Hat, Inc.
```